### PR TITLE
handle "Invalid Date" in show

### PR DESCRIPTION
**Pull request by davidchambers**
See #11, ramda/ramda#1576

We should wait until the Ramda pull request is accepted before merging this pull request.

### CODE REVIEW

**Comment by svozza** [comment]
I'm guessing it's safe to use the global `isNaN` function here because the type coercion it does isn't a problem here.

**Comment by davidchambers** [comment]
> I'm guessing it's safe to use the global `isNaN` function because the type coercion it does isn't a problem here, i.e, `valueOf` always returns a number.

Yes. Initially I used `x.valueOf() === x.valueOf()` as the check, but I then remembered that `isNaN` is well behaved when the domain is restricted to values of type `Number`. `isNaN` is a bit clearer in this case.

I just updated this pull request to depend on `ramda@~0.19.1`, so the test for `R.toString`-like string representations is now a one-liner. :)

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "jscs": "2.7.x",
     "jshint": "2.8.x",
     "mocha": "2.x.x",
-    "ramda": "0.18.x",
+    "ramda": "~0.19.1",
```
Comment (albrow) [inline package.json]: Is there a reason we used the `~` syntax instead of `0.19.x`?

Comment (davidchambers) [inline package.json]: Yes. In this case we're depending on behaviour which changed between v0.19.0 and v0.19.1, via ramda/ramda#1576. We need to use at least v0.19.1 to have the "uses `R.toString`-like string representations" tests pass.

**Pull request merged**